code/modules/antagonists/wizard/grand_ritual/grand_rune.dm

```diff
@@ -340,10 +340,16 @@
 	var/pick = show_radial_menu(user, user, options, require_near = TRUE, tooltips = TRUE)
 	if (!pick)
 		return
+	var/datum/grand_finale/picked_finale = picks_to_instances[pick]
+	var/round_time_passed = world.time - SSticker.round_start_time
+	if(picked_finale.minimum_time >= round_time_passed)
```

Comment:
Won't this runtime if you pick `PICK_NOTHING`?

Reply:
yuh